src/commcare_cloud/terraform/modules/network/main.tf

```diff
@@ -290,7 +290,6 @@ resource "aws_security_group" "g2-access-sg" {
   }
```

Comment:
it should be `dimaig-access-sg` instead `g2-access-sg`

Reply:
Suggestion on the private route table.
can we mark the private route table as default so all the subnets created in future will go directly to the default route table (private) or you see any downfall of that?